marco-themes/HighContrast/metacity-theme-1.xml

```diff
@@ -72,6 +73,9 @@
 </draw_ops>
 
 <draw_ops name="menu_button">
+  <rectangle color="gtk:bg[NORMAL]" filled="true"
```

Comment:
Using fg color makes the arrow visible for me.

Reply:
But it makes it invisible for me with compiz, i.e. metacity rendering: it's black on black
(and drawing a black rectangle is useless then anyway)

Reply:
Same with using metacity itself here, still wondering why we have a differnt here.
The arrow needs to be draw explicit.
But using a black background seems to be ok here to match previous state before the PR.

Reply:
I meant a different to  our default WindowsManager.